content/en/real_user_monitoring/installation/_index.md

```diff
@@ -16,16 +19,51 @@ To set up Datadog Real User Monitoring solution:
 
 1. On the [Real User Monitoring page][1], click the **New Application** button.
 2. Fill out  Application Details, and click **Generate Client Token**. This automatically creates a `clientToken` and an `applicationId` for your application.
-3. Paste the [generated code snippet](#generated-code-snippet) into the head tag (in front of any other script tags) of every HTML page you want to monitor in your application.
-    **Note**: Including the script tag higher and synchronized ensures Datadog RUM can collect all performance data and errors.
+3. Setup the Datadog browser rum library [through NPM](#npm-setup) or paste the [generated code snippet](#bundle-setup) into the head tag.
 4. Deploy the changes to your application. Once your deployment is live, Datadog starts collecting events from your user's browsers.
 5. Visualize the [data collected][2] in Datadog [out of the box Dashboards][3].
 
 **Note**: Your application shows up on the application list page as "pending" until Datadog starts receiving data.
 
-### Generated code snippet
+## NPM Setup
 
-Paste the generated code snippet into the head tag (in front of any other script tags) of every HTML page you want to monitor in your application.
+After adding [`@datadog/browser-rum`][4] to your `package.json` file, initialize it with:
+
+{{< tabs >}}
+{{% tab "US" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum'
+
+datadogRum.init({
+  applicationId: '<DATADOG_APPLICATION_ID>',
+  clientToken: '<DATADOG_CLIENT_TOKEN>',
+  datacenter: 'us',
+  resourceSampleRate: 100,
+  sampleRate: 100
+})
+```
+
+{{% /tab %}}
+{{% tab "EU" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum'
+datadogRum.init({
+  applicationId: '<DATADOG_APPLICATION_ID>',
+  clientToken: '<DATADOG_CLIENT_TOKEN>',
+  datacenter: 'eu',
+  resourceSampleRate: 100,
+  sampleRate: 100
+})
```

Comment:
since sample rates are optional, IMO we could document them only in the advanced section

Reply:
🤔 it's true they are optional, but as i understood it `sampleRate` is a common question support gets, so displaying it here with the default value is a good and risk-free way to give one more hint to our users. but no strong opinions tbh.